docs/rules/generator-star-spacing.md

```diff
@@ -59,7 +59,7 @@ The rule takes one option, an object, which has two keys `before` and `after` ha
 The default is `{"before": true, "after": false}`.
 
 ```json
-"generator-star-spacing": ["error", {"before": false, "after": true}]
+"generator-star-spacing": ["error", {"before": true, "after": false}]
```

Comment:
It looks like this was intended to be a general example of how to configure it, rather than showing another example of the default configuration. Perhaps something along the lines of:

```md
An example configuration:
"generator-star-spacing": ["error", {"before": true, "after": false}]
```

We'll probably want a comment before the shorthand configuration on line 72 as well.